bash/relver

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# relver: use git to set z390.properties version details
+
+set -e              # exit on all errors
+cd $(dirname $0)    # set to directory of script
+cd ..
+echo "version=$(git describe)" > z390.properties
```

Comment:
Line 8 requires that anyone running bldjar, which invokes relver, must (1) be using git, (2) be running bldjar from within the correct git repository. A version of z390 zip downloaded by "Joe User" only requires Java: (1) JRE 1.8+ if the user only intends to run z390; (2) JDK 1.8+ if the user intends to rebuild the z390 jar file (by running bldjar). There cannot (and should not) be a git requirement. Don's recent email illustrates the problem.

Reply:
The last line completely replaces z390.properties. Right now it only has one line, so this is okay. However, if addtional properties are added at a later date, this will need to be changed.

I'd still like to find a non-git way to set the version property.

Reply:
Yes for now. If someone comes up with other properties they want to include then we can change the process then. We also may never do that so didn't want to over engineer it. Just enough to have it working.

Reply:
A non git approach is possible. Let me know what you are thinking.